jobs/build-arch.Jenkinsfile

```diff
@@ -316,7 +316,7 @@ lock(resource: "build-${params.STREAM}-${basearch}") {
 
         // Upload to relevant clouds
         // XXX: we don't support cloud uploads yet for hotfixes
-        if (uploading && !pipecfg.hotfix) {
+        if (uploading && !pipecfg.hotfix && !stream_info.skip_cloud_uploads) {
```

Comment:
I wasn't sure if `!stream_info.skip_cloud_uploads` would work in the case that it's undefined. But we clearly assume it does elsewhere and haven't had any issues, so simplified this!